wfsim/strax_interface.py

```diff
@@ -13,23 +13,28 @@
 export, __all__ = strax.exporter()
 __all__ += ['instruction_dtype', 'truth_extra_dtype']
 
-instruction_dtype = [
-    ('event_number', np.int32),
-    ('type', np.int8),
-    ('time', np.int64),
-    ('x', np.float32),
-    ('y', np.float32),
-    ('z', np.float32),
-    ('amp', np.int32),
-    ('recoil', '<U2')]
+
+#recoil refers to 1:ER, 2=NR, 3=Alpha
+instruction_dtype = [(('Waveform simulator event number.', 'event_number'), np.int32),
+             (('Quanta type (S1 photons or S2 electrons)', 'type'), np.int8),
+             (('Time of the interaction [ns]', 'time'), np.int64),
+             (('End time of the interaction [ns]', 'endtime'), np.int64),
+             (('X position of the cluster[cm]', 'x'), np.float32),
+             (('Y position of the cluster[cm]', 'y'), np.float32),
+             (('Z position of the cluster[cm]', 'z'), np.float32),
+             (('Number of quanta', 'amp'), np.int32),
+             (('Recoil type of interaction.', 'recoil'), np.int),
+             (('Energy deposit of interaction', 'e_dep'), np.float32),
+             (('Eventid like in geant4 output rootfile', 'g4id'), np.int32)
```

Comment:
Cool, thanks. Can we also add [this guy](https://github.com/XENONnT/epix/blob/master/epix/io.py#L164)?

Reply:
Sure